tests/test-workflows.yml

```diff
@@ -2,7 +2,102 @@
   command: arborator --help
 
 - name: Basic arborator test
+  tags:
+    - basic
   command: arborator --profile tests/data/profile.tsv --metadata tests/data/metadata.tsv --config tests/data/config.json --outdir results --thresholds 10,9,8,7,6,5,4,3,2,1
+  stdout:
+    must_not_contain:
+      - "parameter unrecognized"
+  files:
+    - path: "results/cluster_summary.tsv"
+      contains:
+        - "OutbreakID"
```

Comment:
Could you add checks for other data in this file. Specifically the summary info for the metadata columns.